functional_test/build.gradle

```diff
@@ -77,6 +77,13 @@ dependencies {
     }
     testImplementation("com.ning:async-http-client:1.9.40")
     testImplementation("javax.ws.rs:javax.ws.rs-api:2.1")
+
+    //Webservices updates for jakarta ee 9
+    implementation('jakarta.jws:jakarta.jws-api:3.0.0')
```

Comment:
I would prefer the 'implementation' dependencies all be together at top but that could be a nit.